XCUITests/ActivityStreamTest.swift

```diff
@@ -130,6 +130,7 @@ class ActivityStreamTest: BaseTestCase {
 
     func testTopSitesRemoveAllExceptPinnedClearPrivateData() {
         navigator.goto(BrowserTab)
+        waitForNoExistence(app.progressIndicators.element(boundBy: 0))
```

Comment:
I have not used this a lot...not sure how stable is....what about using `waitUntilPageLoad` or `waitForValueContains(app.textFields["url"], value: whateverValueInAddressBar)` just if the one you used gives issues...to have more options in mind.

Reply:
This test was initial failing intermittently. This was the only way to get it working 5 out of 5 times without failure...`waitUntilPageLoad` didn't work.

I could try `waitForValueContains(app.textFields["url"], value: whateverValueInAddressBar)` locally and if it's working flawless I could change it. Sounds good Isabel?

Reply:
yeah, sure! in the past I also had issues with the `waitForPageLoad` so better to try with other options... 🤞